src/H5Pdxpl.c

```diff
@@ -473,6 +477,13 @@ H5P__dxfr_reg_prop(H5P_genclass_t *pclass)
                            NULL) < 0)
         HGOTO_ERROR(H5E_PLIST, H5E_CANTINSERT, FAIL, "can't insert property into class")
 
+    /* Register the actual selection I/O mode property */
+    /* (Note: this property should not have an encode/decode callback -QAK) */
```

Comment:
Should probably remove Quincey's initials here. You can add your own if you want, or just leave it unsigned

Reply:
Fixed.